build.sh

```diff
@@ -143,11 +149,12 @@ pixman_dependencies=""
 cairo_dependencies="zlib png pixman"
 xml_dependencies="zlib iconv"
 sqlite_dependencies=""
-openslide_dependencies="png jpeg tiff openjpeg glib gdkpixbuf cairo xml sqlite"
+openslide_dependencies="libzip png jpeg tiff openjpeg glib gdkpixbuf cairo xml sqlite"
 openslidejava_dependencies="openslide"
 
 # Build artifacts
 zlib_artifacts="zlib1.dll"
+libzip_artifacts="libzip.dll"
```

Comment:
For now, go ahead and use `zip_open()`, but we may need to revisit that eventually.  There are two reasons we might want to use `zip_source_function()` instead, one subtle (`O_CLOEXEC`) and one performance-related (handle cache).